packages/libcomps/libcomps.spec

```diff
@@ -15,17 +15,17 @@
 # Our EL8 buildroots default to Python 3.8, but we also need a 3.6 build of libcomps
 # to make dnf happy
 %if 0%{?rhel} == 8
-%bcond_without python36
+%bcond_without python39
 %else
-%bcond_with python36
+%bcond_with python39
 %endif
```

Comment:
this will break installs, you need a 3.6 build on el8 to not to break system tools that use 3.6 and need libcomps

Reply:
So we will need to have a built and publish version of libcomps for Pulp 3.18? Should I add the python36 devel to our buildroot?